sherpa/stats/__init__.py

```diff
@@ -793,8 +793,11 @@ class Chi2XspecVar(Chi2):
     bin.
 
     The calculation of the variance is the same as `Chi2DataVar`
-    except that if the number of counts in a bin is less than 1
-    then the variance for that bin is set to 1.
+    except that if the number of counts in a bin is less than 1 then
+    the variance for that bin is set to 1. Note that this does not
+    handle background subtraction, so to match XSPEC extra logic is
```

Comment:
This statement is a bit misleading. This statistics is for background subtracted data and the only issue is that for the bins with 0 counts this will use this ad hoc correction. Maybe we do need to stress that for low counts data there should be no subtraction and the applied statistics should be cstat or wstat.

Reply:
Any statement here is mis-leading. However, this class, as written, **DOES NOT** handle background subtraction. We only use this class to calculate the errors when given n counts. There is no concept of background subtraction here (and that is, in part, one of the issues, but it is a separate issue). So, all I want to say is that using Chi2XspecVar with background subtracted data relies on other logic - which is in sherpa.asto.data.DataPHA.get_staterror - to get the correct answer.

I was going to say that most people are not going to see this text, but unfortunately anyone who has done 

```python
set_stat('chi2xspecvar')
```

then will see this information with

```python
get_stat()
```

Reply:
I just looked at the screen output with `get_stat` and it would be less confusing if you can just remove the last sentence from this description.

Reply:
I was thinking about this sentence:
```Note that this does not
    handle background subtraction, so to match XSPEC extra logic is
    present in sherpa.astro.data.DataPHA.get_staterror to handle that
    case.```